common/src/main/java/com/wynntils/functions/GuildFunctions.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright © Wynntils 2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.functions;
+
+import com.wynntils.core.components.Models;
+import com.wynntils.core.consumers.functions.Function;
+import com.wynntils.core.consumers.functions.arguments.FunctionArguments;
+import com.wynntils.models.players.type.GuildRank;
+import com.wynntils.utils.type.CappedValue;
+
+public class GuildFunctions {
+    public static class CappedGuildLevelProgressFunction extends Function<CappedValue> {
+        @Override
+        public CappedValue getValue(FunctionArguments arguments) {
+            return Models.Guild.getGuildLevelProgress();
+        }
+    }
+
+    public static class GuildLevelFunction extends Function<Integer> {
+        @Override
+        public Integer getValue(FunctionArguments arguments) {
+            return Models.Guild.getGuildLevel();
+        }
+    }
+
+    public static class GuildMemberFunction extends Function<Boolean> {
+        @Override
+        public Boolean getValue(FunctionArguments arguments) {
+            return !Models.Guild.getGuildName().isEmpty();
+        }
+    }
```

Comment:
Is this strictly needed? You could use the GuildName function here. Generally, we don't want to add such functions that essentially do the same, as 2-3 functions chained together.

Reply:
Is there a way to return a boolean value if a string is empty or insert literals into the expression? Cause I couldn't find any functionaility in this regard.

Reply:
Yes, you would use `eq_str(guild_name;"")` to check for no guild

Reply:
ah ty